### DISCUSSION TITLE
WPF - 11th Gen Intel Graphics Stops rendering

### DISCUSSION THREAD

**Discussion by Arjunender** [cefsharp/CefSharp#3587] [General]
Hi,

Not able to click a button or access textbox or any object on page and page not rendering properly

The same is working in other system

Please find the attachments

1. system info
2. webpage not rendered properly

![ambika sysdetails](https://user-images.githubusercontent.com/14039139/120216649-eb8f2380-c254-11eb-927a-c7726f8c6412.png)
![bug](https://user-images.githubusercontent.com/14039139/120216655-ed58e700-c254-11eb-9ed0-03b7a9de6001.png)

Thanks

**Comment by amaitland** [comment]
Please edit your original question and provide answers to the questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

**Reply by amaitland** [reply]
If you are using WPF then there is a known issue with 11th Gen iris graphics cards. 

WPF browser stops redrawing when running on Intel Iris Xe Integrated GPU(11th Gen). The issue has been reported to the WPF team see https://github.com/dotnet/wpf/issues/3817

**Reply by amaitland** [reply]
https://www.intel.com/content/www/us/en/support/articles/000058136/graphics/graphics-for-11th-generation-intel-processors.html
